### DISCUSSION TITLE
Only allow dark mode?

### DISCUSSION THREAD

**Discussion by iYonga** [DirectoryLister/DirectoryLister#609] [Help & Support] [answered]
Is allowing only the Dark Mode possible? If so, how?

**Comment by PHLAK** [comment, marked as answer]
This is not possible. Light/dark mode is an end-user choice. The theme will default to the user's system theme though.

**Comment by lunofe** [comment]
For anyone interested in this...

1. Open `/app/views/layouts/app.twig`
2. Find `<div class="{{ theme }}" x-data="application" x-effect="storeThemeCookie(theme)" :class="{ 'dark': theme === 'dark' }">` and replace it with `<div class="dark" x-data="application">` to set the default theme to dark and disable (not hide) the theme toggle
3. To hide the theme toggle _(one of them is theoretically sufficient)_:
      a) Open `/app/views/components/theme-toggle.twig` and remove all of its contents (not the file itself if you don't remove the following part as well)
      b) Open `/app/views/components/header.twig` and remove `<div class="flex items-center justify-center w-6"> {% include 'components/theme-toggle.twig' %} </div>`

Dark mode accomplished.

**Reply by zaowens** [reply]
Very helpful comment, thank you.

**Reply by visualblind** [reply]
@lunofe Thanks for posting this. This should have been accepted as the answer instead of @PHLAK's answer.

@PHLAK for being the developer of this software, I would have expected more knowledge of it.
